src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStages.tsx

```diff
@@ -28,6 +35,14 @@ const MSG = defineMessages({
     id: 'dashboard.ExpenditurePage.Stages.StreamingStages.notStarted',
     defaultMessage: 'Not started',
   },
+  openModal: {
+    id: 'dashboard.ExpenditurePage.Stages.StreamingStages.openModal',
+    defaultMessage: 'Open modal',
+  },
+  tooltipOpenModalText: {
+    id: 'dashboard.ExpenditurePage.Stages.StreamingStages.tooltipOpenModalText',
+    defaultMessage: 'Cancel streaming payment',
```

Comment:
As per @Julianahlk's comment here: https://github.com/JoinColony/colonyDapp/pull/3886#discussion_r985922013, shouldn't this be capitalized too?

Reply:
i have changed it here too